cogs/torrents.py

```diff
@@ -0,0 +1,66 @@
+import os
+import discord
+import requests
+import json
+from discord.ext import commands
+
+
+def load_json(token):
+    with open('./config.json') as f:
+        config = json.load(f)
+    return config.get(token)
+
+
+def setup(client):
+    client.add_cog(Torrents(client))
+
+
+def get_magnet(search: str) -> discord.Embed:
+    """Get a magnet link for a torrent"""
+
+    failure = discord.Embed(
+        title='Unable to find torrent',
+        url=f'https://www.thepiratebay.org/search/{search}',
+        description='No Magnet Found'
+    )
+    failure.set_footer(text='Better luck next time!')
+
+    try:
+        response = requests.get(f'https://apibay.org/q.php?q={search}')
+    except Exception:
```

Comment:
This should be more explicit

Reply:
Right, are we just logging via stdout?

Reply:
I don't do much logging. Don't tell anyone.